packages/editor/src/components/block-switcher/index.js

```diff
@@ -61,15 +61,11 @@ export class BlockSwitcher extends Component {
 				return null;
 			}
 			return (
-				<Toolbar>
-					<IconButton
-						disabled
-						className="editor-block-switcher__no-switcher-icon"
-						label={ __( 'Block icon' ) }
-					>
+				<div className="components-toolbar" aria-hidden="true">
```

Comment:
We are using the class of a component outside of it in another element. I guess in this case we should use a different class and add the minimum necessary styles to have a similar UI.

Reply:
As mentioned this should be:
```suggestion
				<div className="components-block-switcher" aria-hidden="true">
```

As mentioned by Jorge you'll need to duplicate some styles from `components-toolbar`, I'd imagine, to get this to work.

Reply:
@jorgefilipecosta @tofumatt All set here